src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/WithTransferSender.cshtml

```diff
@@ -39,12 +39,17 @@
 
                 </tbody>
             </table>
+        }
+        else
+        {
+    <p class="govuk-body govuk-!-font-weight-bold">You currently have 0 apprentices with transfer sending employers</p>
+        }
 
-        </div>
     </div>
+
 </div>
 
 @section Back
-{     
+{
     <a id="back-link" class="govuk-back-link" href="@Url.Action("Cohorts", "Cohort", new { Model.AccountHashedId})">Back</a>
```

Comment:
<div class="das-js-back-link"></div>

Reply:
This change has been done on another branch. Its in a PR.